flopy/utils/binaryfile.py

```diff
@@ -710,9 +696,10 @@ def reverse(self, filename: Optional[os.PathLike] = None):
         # get total number of records
         nrecords = len(self)
 
-        # open backward file
-        with open(filename, "wb") as fbin:
-            # loop over head file records in reverse order
+        # loop over head file records in reverse order and write temp file
+        temp_file = tempfile.NamedTemporaryFile()
+        temp_file_path = Path(temp_file.name)
+        with open(temp_file_path, "wb") as f:
```

Comment:
This throws a permission error on my Windows 10 machine.

See also: https://stackoverflow.com/questions/23212435/permission-denied-to-write-to-my-temporary-file

Reply:
Pesky platform differences.. Thanks for the pointer